### ISSUE TITLE
Accessibility: MAS40B: Inspect: Localized control type property is wrong for "Skip to Content" button

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#32]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

Testing Environment:
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open [https://code.visualstudio.com/docs/introvideos/overview](https://code.visualstudio.com/docs/introvideos/overview) in EDGE
2. Navigate to "Skip to Content" button
3. Observe the "Localized Control Type" property in Inspect
**_Same issue is observed with the links:_**
[https://code.visualstudio.com/docs
](https://code.visualstudio.com/docs
)https://code.visualstudio.com/Download
https://code.visualstudio.com/blogs
https://code.visualstudio.com/updates
https://code.visualstudio.com/docs/supporting/faq
https://code.visualstudio.com/insiders
https://code.visualstudio.com/docs/editor/accessibility

**Actual Result:**
The "Localized Control Type" property is link instead of button
**_Narrator Behaviour:_**
Narrator announces as "Skip to Content link"

**Expected Result:**
The "Localized Control Type" property should be button

**User Impact:**
If the control type is link, it will be confusing for a screen reader user and the user will understand that the link will redirect to some other page.

**Screenshots:**
![localized control type is link for skip to content button](https://cloud.githubusercontent.com/assets/24825441/21709733/6744f430-d409-11e6-837d-5bfbf370571c.png)
